config/manager/kustomization.yaml

```diff
@@ -1,2 +1,7 @@
 resources:
 - manager.yaml
+
+images:
+- name: controller
+  newName: quay.io/redhat-appstudio/project-controller
```

Comment:
I think this should reside in the infra-deployments repo, not here.

Reply:
I checked image-controller repo and they have it here and when I give this commit sha in infra-deployments the kube-linter errors are resolved.

Reply:
@ifireball This is the [PR](https://github.com/redhat-appstudio/infra-deployments/pull/3456) in infra-deployments where the commit sha from here would be updated automatically after we have release plan admission setup.

Reply:
This name configuration is specific to the Red Hat deployment, it shouldn't be here. It didn't use to matter for the repos in redhat-appstudio, but now in knoflux-ci it does. We need to make an effort to put the Red Hat specific stuff in infra-deployments if we can.